proposals/lip-update_genesis_block_schema_and_processing.md

```diff
@@ -0,0 +1,169 @@
+All of these checks are stateless since the state of the blockchain is yet to be initialized. 
+The key differences as compared to [the validation for the rest of the blocks][BlockLIPvalidation] in a blockchain are that there is no specific size limit for the genesis block object and that the payload must be empty, i. e., the genesis block should not contain any transaction.
+
+#### Genesis state Initialization
+
+At this stage, the genesis state initialization logic for all registered modules is executed. 
+For this purpose, registered modules can specify processing logic considering the information in their corresponding entry of the `assets` property. 
+Typically, modules will perform format and data consistency checks of their respective element in `assets` and then initialize their state according to the data provided in it. 
+However, modules should not call protocol logic of other modules as the state of the respective module may not be initialized.
+
+#### Genesis state finalization
+
+At this stage, the genesis state finalization logic for all registered modules is executed. 
+As part of the genesis state finalization logic modules may call exposed functions from other modules to cross-check the state information integrity and/or complete their own state. 
+When this step is completed, there should be a valid initial state for our blockchain ready to process state transitions implied by the next block.
+
+#### Result verification of the genesis block
+
+Finally, the verifications for block header properties are performed for which access to the state store is required. 
+For example, in the case of `stateRoot`, it requires the genesis state to be final before being checked.
+
+## Specification
+
+In this section, we specify the schema of the genesis block and its validity and execution rules.
+
+### Constant
+
+| Name          | Type    | Value       | Description |
+| ------------- |---------| ------------| ------------|
+| `EMPTY_HASH`  | bytes   | SHA-256("") | Hash of empty bytes.|
+
+### Processing stages of the genesis block, block assets, and block header
+
+As introduced in [the Rationale section](#processing-of-the-genesis-block), the processing of the genesis block is performed in four different stages:
+
+* _Static validation of the genesis block_: The stateless checks to ensure the structure of the genesis block are performed. 
+Also, the properties in the block header that do not require access to the state store are checked. 
+These checks are defined in the sections below.
+* _Genesis state initialization_: The genesis state initialization logic for the registered modules is executed. 
+* _Genesis state finalization_: The genesis state finalization logic for the registered modules is executed.  
+* _Result verification of the genesis block_: The block header properties that require access to the state store are verified as specified below.
+
+### Genesis block
+
```

Comment:
Here, I think we could ass another header. Below we have 2 sections:
- Header of the genesis block
- Assets property of the genesis block
So this section could be called 
- Transactions property of the genesis block

This way, we cover the 3 properties of the block in a symmetric manner.

Reply:
I prefer to keep it as it is so it is the same as [the other block schema LIP already merged](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md).